# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/webidl/WebTaskScheduling.webidl
# Commit: be6c223e63f0
# Full Hash: be6c223e63f0a0c8ef2428df42321d59e4c0f7c6
# Author: Sean Feng <sefeng@mozilla.com>
# Date: 2025-02-25 09:30:56
# Regressor Bug: 1920115
# File Overlap Count: 2
# Description:
#   Bug 1920115 - Implement scheduler.yield r=jjaschke,dom-core,webidl,smaug
#   
#   Spec: https://wicg.github.io/scheduling-apis/#dom-scheduler-yield
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D234232
# ==============================================================================

diff -r 121bf804d2a6 -r be6c223e63f0 dom/webidl/WebTaskScheduling.webidl
--- a/dom/webidl/WebTaskScheduling.webidl	Mon Feb 24 19:31:26 2025 +0000
+++ b/dom/webidl/WebTaskScheduling.webidl	Mon Feb 24 19:31:27 2025 +0000
@@ -31,6 +31,9 @@
     SchedulerPostTaskCallback callback,
     optional SchedulerPostTaskOptions options = {}
   );
+
+  [BinaryName="yieldImpl"]
+  Promise<undefined> yield();
 };
 
 dictionary TaskControllerInit {